apps/entity-service/src/entities/dto/site.dto.ts

```diff
@@ -54,17 +54,20 @@ export class SiteLightDto extends EntityDto {
   @ApiProperty({ nullable: true })
   name: string | null;
 
-  @ApiProperty({
-    nullable: true,
-    description: "The associated project name"
-  })
-  projectName: string | null;
-
   @ApiProperty()
   createdAt: Date;
 
   @ApiProperty()
   updatedAt: Date;
+
+  @ApiProperty({ type: Project })
+  project: Project;
```

Comment:
Lots of problems here:
* We'd never want to include the full project on another entity's DTO; that's a pattern from PHP we're ditching in v3. Fetching a couple of fields (like UUID and name) is OK, but if an element on the page needs the full project, it can use the project GET connection to pull it.
* A model cannot itself be an ApiProperty - it has to be a DTO
* I still think the project information isn't needed on the light DTO. Did you find a case where that's incorrect? Also applies to `projectUuid` below.

Reply:
Agree, we are removing project as a entity from the dto

Reply:
Actually the list of sites in admin do require the `projectName` property, so that one will stay on the light dto.